implementation/interfacing_with_emlib.tex

```diff
@@ -164,32 +164,226 @@ \subsection{Naming Conventions}
 \label{lst:enum_naming_rust}
 \end{listing}
 
+\subsection{The Bindings Libraries}
+\label{ssub:the_bindings_library}
+
+We have written many example applications in Rust that are based on bindings for many of the {\gecko}'s peripherals, these examples have been a driving factor for defining new bindings.
+As an example, if we were to need some functionality for the \gls{adc}, we would define these bindings alongside the development of the examples.
+In this way, the bindings have been defined incrementally during the development of either the examples that demonstrate our libraries, or the applications described in \todo{Refer to sections about circle-game and sensor-tracker}.
+
+\subsubsection{Emlib}
+
+It has not been a priority of this project to fully support all of the {\gecko}'s peripherals or provide bindings for the entire {\emlib}.
+The bindings have been developed because we have needed functionality from the peripherals, not the other way around.
+\autoref{tab:emlib_peripheral_bindings} summarizes a number of modules from {\emlib} that we have written bindings for, and why we wanted these bindings.
+A list of examples that demonstrates how the bindings work is shown in \autoref{tab:emlib_examples}, these are either written from scratch, or directly ported from {\emlib}'s examples.
```

Comment:
As we are emphasising on the examples + project being the driving factors, perhaps they should be presented first?